.travis.yml

```diff
@@ -1,10 +1,21 @@
+git:
+  depth: 10
+
 language: node_js
+
 node_js:
   - "4"
-script: make travis
+
+script: node make travis
```

Comment:
Because of how the `Makefile` setup, these are identical. So yeah, I'm good either way. :D